lib/wallet/src/Cardano/Wallet/IO/Delegation.hs

```diff
@@ -0,0 +1,293 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+
+-- |
+-- Copyright: © 2018-2020 IOHK
```

Comment:
I would argue it should be adjusted to realities ;-)

Reply:
Fair enough. 😂 Done.